lib/node_modules/@stdlib/math/base/special/logitf/src/main.c

```diff
@@ -0,0 +1,50 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/logitf.h"
+#include "stdlib/math/base/assert/is_probability.h"
```

Comment:
@vivekmaurya001 We'll need to use `math/base/assert/is_probabilityf` here, which isn't implemented as of now.

Reply:
should i implement is_probabilityf first ?